### PR TITLE
added information on catering

### PR DESCRIPTION
**Pull request by raidancampbell**

### CODE REVIEW

**Comment by mtbentley** [comment]
Looks good :+1:

Maybe also include a list of places we like to order from with specific notes about each? (eg, for Potbelly get a greater variety than what they suggest)

**Comment by raidancampbell** [comment]
Einen Moment, Bitte.

**Comment by raidancampbell** [comment]
Waiting for someone to tell me whether JJ's gives napkins & plates with their orders

File: restaurant_catering_tips.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+----------------
+
+- Surprisingly cheap, sandwiches are $5 each for cheapest tier
+- Easy to scale, chips/cookie are each available at higher tiers.  Assorted by default
+- Variety is important.  Most sandwiches are effectively a "pile of meat"
+- Provides toppings on the side
+- Provides plates/napkins
+- Offers chili and salad as well.  Serving/consuming utensils provided
+- Ordering is pretty painless and straightforward
+- We're routinely they're largest single customer
```
Comment (thomasrussellmurphy) [inline restaurant_catering_tips.rst]: their

**Comment by thomasrussellmurphy** [comment]
JJ's does not provide utensils/plates/napkins, we just usually have enough around already.

File: restaurant_catering_tips.rst (hunk 1 above)
Comment (raidancampbell) [inline restaurant_catering_tips.rst]: Grah.  I hate it when people have bad grammar.  Thanks for the catch.

**Comment by raidancampbell** [comment]
Fixed both issues.  This is now ready for merging IMO.

**Comment by thomasrussellmurphy** [comment]
Looks like enough of a thing.

**Comment by brenns10** [comment]
Beautiful.

**Pull request merged**